### DISCUSSION TITLE
Notify: Ability to customize closeBtn color

### DISCUSSION THREAD

**Discussion by lucasdavidferrero** [quasarframework/quasar#15304] [Ideas / Proposals] [answered]
Would be nice if we provide the ability to set custom color to the `closeBtn`. 

User suggestion: https://stackoverflow.com/questions/75183452/quasar-notify-how-to-change-close-button-color/75185405#75185405

**Comment by rstoenescu** [comment, marked as answer]
The `closeBtn` is there for convenience. It's a shortcut to automatically add an action button.
If you need customization, directly add an action instead. It would be stupid to overcomplicate a piece of the API which is a convenience with a duplicated functionality.
